config/dependencies.yml

```diff
@@ -49,15 +49,15 @@ gax:
     version: '0.12.5'
     next_version: '0.13.0'
   ruby:
-    version: '0.4.1'
+    version: '0.5.0'
```

Comment:
https://github.com/googleapis/gax-ruby/pull/38 bumps the version to `0.4.4`, not `0.5.0`. Is this in anticipation of more changes?